checks-superstaq/checks_superstaq/pylint_import_annotations.py

```diff
@@ -0,0 +1,69 @@
+"""Check for whether python file has from __future__ import annotations as the first import."""
+
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING
+
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+
+if TYPE_CHECKING:
+    from astroid import nodes
+    from pylint.lint import PyLinter
+
+
+class ImportAnnotationsChecker(BaseChecker):
+    """Checker for whether future annotations is imported in python file."""
+
+    name = "import-future-annotations"
+    msgs = {
+        "W6071": (
```

Comment:
is there a reason not to use W6063?

Reply:
If that plugin is ever extended, then it would be taken. Also, I was trying to replicate the grouping structure that Pylint has with their plugins where each plugin starts at `xxx1` and expands from there. However, since this is an internal plugin, it is reasonable to use `W6063` to group together all our plugins. Which one should I go with?